### DISCUSSION TITLE
Heroku app help

### DISCUSSION THREAD

**Discussion by LeParan0** [MrMissx/Telegram_Forwarder#12] [Q&A]
Hello,

I try running the app on Heroku, but i think i doing something wrong. The config Vars are configured, the app is "build succeeded" and i run the command "pip3 install -r requirements.txt".
But i think i passed something.
Can you help me ?
Thanks.

**Comment by LeParan0** [comment]
Sorry i made a mistake with the configuration of my bot and groups in telegram.

**Reply by deepakksharma00079** [reply]
now is your bot working?

**Reply by LeParan0** [reply]
> now is your bot working?

Yes, i dont try right now, but last time was ok (~1 month ago)
